Create ui-c8y-1023-23-1-prevent-duplicate-property-names-in-propert-10855-graft-release-cd-11005.md (#4178)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-23-1-prevent-duplicate-property-names-in-propert-10855-graft-release-cd-11005.md

---
date: ""

> title: prevent duplicate tenant custom property names in Properties Library

Reviewer: ```suggestion
title: Duplicate name validation added to prevent duplicate tenant custom property names
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-65556
version: 1023.23.1
---

> Users could save multiple properties with the same name in the Properties Library (tenant custom properties). Only one would display in the tenant custom properties tab. Duplicate name validation has been added to the name field, showing an error when a property with the same name already exists and disabling the save button until a unique name is provided.

Reviewer: ```suggestion
Previously, users could save multiple properties with the same name in the tenant's custom properties, with the result that only one property with that name was displayed on the tenant's **Custom properties** tab. Duplicate name validation has been added to the name field, showing an error when a property with the same name already exists and disabling the save button until a unique name is provided.
```

> Previously, users could save multiple properties with the same name in the tenant's custom properties, with the result that only one property with that name was displayed on the tenant's **Custom properties** tab. Duplicate name validation has been added to the name field, showing an error when a property with the same name already exists and disabling the save button until a unique name is provided.

Reviewer: ```suggestion
Previously, users could save multiple properties with the same name in the property library (tenant's custom properties), resulting in only one property with that name being displayed on the tenant's **Custom properties** tab. Duplicate name validation has been added to the name field, showing an error when a property with the same name already exists and disabling the save button until a unique name is provided.
```

> Previously, users could save multiple properties with the same name in the property library (tenant's custom properties), resulting in only one property with that name being displayed on the tenant's **Custom properties** tab. Duplicate name validation has been added to the name field, showing an error when a property with the same name already exists and disabling the save button until a unique name is provided.

Reviewer: ```suggestion
Previously, users could save multiple properties with the same name in the property library (tenant custom properties), resulting in only one property with that name being displayed on the tenant's **Custom properties** tab. Duplicate name validation has been added to the name field, showing an error when a property with the same name already exists and disabling the save button until a unique name is provided.
```
